### DISCUSSION TITLE
Personal Account to Organisation conversion error

### DISCUSSION THREAD

**Discussion by adrianvdh** [community/community#67465] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

Hi there

I registered a personal account named Serveox.

I then proceeded to convert this account into a Github Organisation. I saw all the warnings and accepted the terms.

The process started, and then I was instantly signed out. No emails were sent to me thereafter. I cannot seem to access this Organisation, and I'm not a member of this org either. How can I regain access to it??

Please will you assist me?

https://github.com/Serveox

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @Adrianvdh ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate post, and we ask that you no longer cross-post. Please note: the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by adrianvdh** [reply]
Hi @queenofcorgis,

My sincere apologies for posting in this forum. I will respect your Community Guidelines and pay more attention to their details.

May I please ask that someone from the GitHub Support team graciously attend to my support issue?

My company intends to migrate to GitHub and enter a paid plan, however, I am currently locked out of the organization I've recently created.

https://github.com/orgs/community/discussions/67463

Thank you kindly
